sashimi-webapp/src/components/editor-viewer/Viewer.vue

```diff
@@ -143,4 +143,39 @@
       width: auto;
     }
   }
+
+  @media print {
+    @page {
+      size: auto;
+      margin: 0;
+    }
+
+    #app > div > div.section.group.content > div:nth-child(2) {
```

Comment:
May need to discuss this line of code later. But for now, let's just use this first.